src/utils/utils_concurrency.h

```diff
@@ -14,7 +14,17 @@
 
 #ifdef _WIN32
 #include <windows.h>
-#else
+
+// Disable warning 28251: "inconsistent annotation for function" thrown in
+// intrin.h, as we do not want to modify this file.
+#pragma warning(push)
+#pragma warning(disable : 28251)
+#include <intrin.h>
```

Comment:
I see that we already include intrin.h in utils_windows_math - maybe we could move this to utils_common.h?

Reply:
Moved to `utils_windows_intrin.h`.
Done